pyproject.toml

```diff
@@ -122,6 +122,11 @@ requires = [
         name = "C API changes"
         showcontent = true
 
+    [[tool.towncrier.type]]
+        directory = "c_api_removal"
+        name = "NumPy 2.0 C API removals"
+        showcontent = true
+
```

Comment:
Do you want to add snippets for the removals in #23916?

Reply:
I wasn't going to, because those have all been effectively removed already (use is always an error)